addon/src/index.js

```diff
@@ -65,7 +67,14 @@ export function setupFactoryGuy(hooks, requestManager) {
     const { owner } = getContext();
     FactoryGuy.setStore(owner.lookup('service:store'));
 
-    const rm = requestManager ? requestManager : new RequestManagerPretender();
+    let DefaultRequestManager;
+    if (interceptor === 'pretender') {
+      DefaultRequestManager = RequestManagerPretender;
+    } else if (interceptor === 'msw') {
+      DefaultRequestManager = RequestManagerMSW;
+    }
+
+    const rm = requestManager ? requestManager : new DefaultRequestManager();
```

Comment:
request manager can be provided to the setupFactoryGuy() function, otherwise it will default to what it set in the config